src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+@import "stars";
+@import "header";
+@import "variables";
```

Comment:
It should be the first import to make the variables visible in the header block. And remove variables from the header